### PR TITLE
OSSMDOC-428: Break upgrade into modules.

### PR DESCRIPTION
**Pull request by JStickler**
Version(s): 4.6 - 4.11

Issue: [OSSMDOC-428](https://issues.redhat.com/browse/OSSMDOC-428)

Link to docs preview: http://file.bos.redhat.com/jstickle/OSSMDOC-428/service_mesh/v2x/upgrading-ossm.html

QE review: pbajjuri20
Peer review: kelbrown20

### CODE REVIEW

**Review by pbajjuri20 — APPROVED** [review]
It looks good to me.

File: modules/ossm-upgrade-20-21-changes.adoc [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+Upgrading the control plane from version 2.0 to 2.1 introduces the following architectural and behavioral changes.
+
+.Architecture changes
+
+Mixer has been completely removed in {SMProductName} 2.1. Upgrading from a {SMProductName} 2.0.x release to 2.1 will be blocked if Mixer is enabled.
+
+[id="ossm-upgrading-differences-behavior_{context}"]
+.Behavioral changes
+
+* `AuthorizationPolicy` updates
```
Comment (kelbrown20) [inline modules/ossm-upgrade-20-21-changes.adoc:17]: Do you think these need `:`. Ex `Authorization Policy updates:`

File: modules/ossm-upgrade-20-21-changes.adoc [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+** With the PROXY protocol, if you're using `ipBlocks` and `notIpBlocks` to specify remote IP addresses, update the configuration to use `remoteIpBlocks` and `notRemoteIpBlocks` instead.
+** Added support for nested JSON Web Token (JWT) claims
```
Comment (kelbrown20) [inline modules/ossm-upgrade-20-21-changes.adoc:19]: I believe this needs a period since

File: modules/ossm-upgrading-from-21-to-22.adoc [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+.Procedure
+
+. Switch to the project that contains your `ServiceMeshControlPlane` resource. In this example, `istio-system` is the name of the control plane project.
++
+[source,terminal]
+----
+$ oc project istio-system
+----
+
+. Check your v2 `ServiceMeshControlPlane` resource configuration to make sure it is valid.
```
Comment (kelbrown20) [inline modules/ossm-upgrading-from-21-to-22.adoc:25]: `make sure` -> `verify`

**Review by kelbrown20 — COMMENTED** [review]
Just some small nits and questions. Otherwise, this looks great! LGTM

**Pull request merged**